src/components/WorkOrder/PrintJobTicketDetails.tsx

```diff
@@ -53,7 +72,7 @@ const PrintJobTicketDetails = ({
                 <path d="M200.955311,28.5221077 C200.298901,30.3958684 199.56337,32.7989483 198.29304,34.3912803 C196.164103,37.0407925 192.887912,37 189.733333,37 L186.048352,37 L186.048352,30.5577263 L187.92967,30.5577263 C188.750183,30.5577263 189.813919,30.6408425 190.389744,30.3550393 C190.880586,30.1115234 191.1663,29.7032331 191.1663,28.7670818 C191.1663,27.7463561 188.054212,19.964052 187.604396,18.7406393 L184,9 L192.681319,9 L196.082051,21.1451774 L196.164103,21.1451774 L199.604396,9 L208,9 L200.955311,28.5221077"></path>
               </g>
               <image
-                src="/assets/images/lbh-logo.png"
+                // src="/assets/images/lbh-logo.png"
```

Comment:
image with no src?

Reply:
`[Type '{ src: String; }' is not assignable to type 'SVGProps<SVGImageElement>](https://stackoverflow.com/questions/59455464/type-src-string-is-not-assignable-to-type-svgpropssvgimageelement)`

Reply:
I have replaced it with an Img tag